_data/faqs.yaml

```diff
@@ -1,76 +1,76 @@
 - question: >-
-    What is the project timeline?
-  answer: >-
-    Development of the repository application began in March 2016 after an intensive Design Phase was undertaken and produced a collection of <a href="https://wiki.duraspace.org/display/hydra/Hydra-in-a-Box+Design+Documents">design documents</a>. A series of project team work cycles and community-based sprints now follow, leading up to several pilots planned for 2017. Watch our <a href="http://hydrainabox.projecthydra.org/blog.html">blog</a> or <a href="https://www.youtube.com/channel/UC1eopKtm-k_f93RTfzHSvMQ">demo videos</a> for updates along the way. More information about the 2017 pilots and service roll-out will be posted here in Fall 2016.
+    What is Hyku?
+  answer: >- 
+    Hyku is the official name of the repository product that is a main deliverable of the Hydra-in-a-Box project. We determined the name in November 2016 and announced it in early December 2016. Prior to that time, we referred to it as “the Hydra-in-a-Box repository” or “Lerna” as a temporary placeholder name. Sometimes we refer to it informally as “HyBox”. In all these cases, we are talking about the same thing: a polished, turnkey, feature-complete repository application product created by the Hydra-in-a-Box project.
```

Comment:
> Sometimes we refer to it informally as “HyBox”

I wonder if we want to commit that to writing. (Or work together to stamp it out. ;) ) Do you think this will help unconfuse folks if they see references to "HyBox?"